### ISSUE TITLE
panic: runtime error: index out of range [0] with length 0

### ISSUE THREAD

**Issue by nvtkaszpir** [arangodb/kube-arangodb#893]
arangodb-operator version: 1.1.5
k8s version: 1.18...1.22

arango operator sometimes crashes with the following message:

```text
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/arangodb/kube-arangodb/pkg/util/k8sutil.GetArangoDBImageIDFromPod(0xc000372000, 0xc00004403c, 0x24)
	/home/ubuntu/.GIT/arango/kube-arangodb/pkg/util/k8sutil/images.go:46 +0x1ef
main.getMyPodInfo.func1(0x2cbbf40, 0xc0004e5288)
	/home/ubuntu/.GIT/arango/kube-arangodb/main.go:357 +0x150
github.com/arangodb/kube-arangodb/pkg/util/retry.retry.func1(0xc000338b40, 0xc000337200)
	/home/ubuntu/.GIT/arango/kube-arangodb/pkg/util/retry/retry.go:75 +0x33
github.com/cenkalti/backoff.RetryNotify(0xc0004e53b8, 0x1f7d0c0, 0xc000338b40, 0x0, 0x0, 0x0)
	/home/ubuntu/.GIT/arango/kube-arangodb/.gobuild/pkg/mod/github.com/cenkalti/backoff@v2.1.1+incompatible/retry.go:37 +0xb8
github.com/cenkalti/backoff.Retry(...)
	/home/ubuntu/.GIT/arango/kube-arangodb/.gobuild/pkg/mod/github.com/cenkalti/backoff@v2.1.1+incompatible/retry.go:24
github.com/arangodb/kube-arangodb/pkg/util/retry.retry(0x0, 0x0, 0xc0004e5430, 0x45d964b800, 0x0, 0x0)
	/home/ubuntu/.GIT/arango/kube-arangodb/pkg/util/retry/retry.go:99 +0xca
github.com/arangodb/kube-arangodb/pkg/util/retry.Retry(...)
	/home/ubuntu/.GIT/arango/kube-arangodb/pkg/util/retry/retry.go:111
main.getMyPodInfo(0x1fefaa0, 0xc000174000, 0xc000046011, 0xf, 0xc00004403c, 0x24, 0x40bdd4, 0x7f4707300748, 0x9, 0x0, ...)
	/home/ubuntu/.GIT/arango/kube-arangodb/main.go:364 +0xf5
main.newOperatorConfigAndDeps(0xc00084a2d0, 0x49, 0xc000046011, 0xf, 0xc00004403c, 0x24, 0x0, 0x0, 0x0, 0x0, ...)
	/home/ubuntu/.GIT/arango/kube-arangodb/main.go:289 +0x13b
main.cmdMainRun(0x2ca7940, 0xc000838300, 0x0, 0x3)
	/home/ubuntu/.GIT/arango/kube-arangodb/main.go:224 +0x689
github.com/spf13/cobra.(*Command).execute(0x2ca7940, 0xc000134050, 0x3, 0x3, 0x2ca7940, 0xc000134050)
	/home/ubuntu/.GIT/arango/kube-arangodb/.gobuild/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x29d
github.com/spf13/cobra.(*Command).ExecuteC(0x2ca7940, 0xc000132120, 0x7, 0xc000114058)
	/home/ubuntu/.GIT/arango/kube-arangodb/.gobuild/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/home/ubuntu/.GIT/arango/kube-arangodb/.gobuild/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
main.main()
	/home/ubuntu/.GIT/arango/kube-arangodb/main.go:151 +0x49
```

**Comment by informalict** [comment]
@nvtkaszpir I will handle it.

**Comment by nvtkaszpir** [comment]
ping

**Comment by informalict** [comment]
https://github.com/arangodb/kube-arangodb/pull/897 is merged.
There is a release 1.2.8 in progress, so today should be available when everything is fine.

**Comment by ajanikow** [comment]
Fixed with 1.2.8
